src/Resources/public/reference-entity/attribute/table/view.tsx

```diff
@@ -6,6 +6,17 @@ import TableAttributeModal from './modal';
 import { connect } from 'react-redux';
 import $ from 'jquery';
 
+export type RightsType = {
```

Comment:
Why not simply `Rights`? I mean its obvious that this is a type...

Reply:
No specific reason. I wanted to prevent confusion about `rights` and `Rights`.